src/tests/test-responses.ts

```diff
@@ -398,6 +398,32 @@ export const checkoutCompleteResponse: CheckoutCompleteResponse = {
   },
 };
 
+export const getVirtualCurrenciesResponseWith3Currencies = {
+  virtual_currencies: {
+    GLD: {
+      balance: 100,
+      code: "GLD",
+      description: "It's gold",
+      name: "Gold",
+    },
+    SLV: {
+      balance: 100,
+      code: "SLV",
+      name: "Silver",
```

Comment:
Intentionally omitting `description` here since it is nullable and may not be included in responses from the backend